scoresheets/CarryMyLuggage.tex

```diff
@@ -3,14 +3,16 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem{100}{Picking up the correct bag}
-	\scoreitem{300}{Following the person to the car}
+	\scoreitem{15}{Perceiving the correct beg (visualize on screen or say which one)}
+	\scoreitem{30}{Picking up the correct bag}
+	\scoreitem{100}{Following the person to the car}
     \scoreitem{50}{Avoiding the crowd of people obstructing the path}
 	\scoreitem{50}{Avoiding the small object on the ground}
 	\scoreitem{50}{Avoiding the hard-to-see object}
 	\scoreitem{50}{Avoiding the area blocked with retractable barriers}
 	
 	\scoreheading{Bonus rewards}
+	\scoreitem{300}{Successful task completion}
```

Comment:
what does this mean? is is to accomplish all the main goal items?

Reply:
yes. Basically if all sub-tasks are completed, one gets bonus points in order to encourage teams to try and complete the entire task instead of going for solving as many sub-tasks as possible.

Reply:
quoting johannes: does this include completing the main goal using deus ex machina? please clarify.
perhaps move this to main goals.
suggestion: give these points for a team solving all main goals without using deus ex machina.

Reply:
quote Rulebook: "Bonuses: No bonus points can be scored when the robot requests help to solve part of a
task that normally would grant a bonus." 
-> So then the Bonus points for task completion wouldn't have been awarded either in my understanding
-> removed it anyway due to the proposed suggestion